specs/helper.js

```diff
@@ -76,21 +75,32 @@ export const overlayAttribute = getModalAttribute('overlay');
 
 const Simulate = TestUtils.Simulate;
 
-const dispatchMockEvent = eventCtor => (key, code) => (element, opts) =>
-  eventCtor(element, Object.assign({}, {
-    key: key, keyCode: code, which: code
-  }, opts));
+const dispatchMockEvent = eventCtor =>
+  (key, code) =>
+    (element, opts) =>
+      eventCtor(
+        element,
+        Object.assign(
+          {},
+          {
+            key: key,
+            keyCode: code,
+            which: code
+          },
```

Comment:
...or change the `code` on the arguments to `keyCode`. 
`{ key, keyCode, which: keyCode }`